.travis.yml

```diff
@@ -21,13 +21,13 @@ install:
   - nvm install stable
   - nvm use stable
   - npm install jsdom
-
+  - pip install --user awscli
 script:
-  - sbt headerCheck test:headerCheck scalastyle sql/flywayMigrate compile test coreJS/test coreJS/fastOptJS
+  - sbt headerCheck test:headerCheck scalastyle sql/flywayMigrate compile test ui/fastOptJS ui/fullOptJS
```

Comment:
We won't run the test cases in JS now?

Reply:
they should run implicitly now with the `aggregate` change